base/sparse/sparsematrix.jl

```diff
@@ -1738,8 +1738,8 @@ for (Bsig, A1sig, A2sig, gbb, funcname) in
         global $funcname
         function $funcname(f::Function, B::$Bsig, A1::$A1sig, A2::$A2sig)
             func       = @get! cache  f  gen_broadcast_function_sparse($gbb, f, ($A1sig) <: SparseMatrixCSC)
```

Comment:
Ew. Unrelated, but hopefully this can become an ordinary higher-order function now like the rest of `broadcast`.